pkg/mathematic/mathematic.go

```diff
@@ -26,6 +26,15 @@ func MaxInt(nums ...int) int {
 	return res
 }
 
+// AbsInt return the absolute value.
+func AbsInt(x int) int {
+	if x < 0 {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498955)